### PR TITLE
15 components/footer

### PR DESCRIPTION
**Pull request by mahmodlte**
Footer fixed

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -1,7 +1,48 @@
 import React from 'react';
-
+import workoutImage from './assets/workoutImage.jpg';
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div
+      className=" sm:h-auto lg:h-64 flex lg:flex-row xl:flex-row flex-col md:flex-col justify-between items-center font-poppins text-secondary"
+      style={{
```
Comment (osamaakb) [inline src/components/Footer/Footer.js:7]: don't use inline styles, replace them with tailwind styles or external css

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -1,7 +1,48 @@
...
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+      }}
+    >
+      <h1 className="text-4xl p-5 select-none">
+        Train to <span className="border-b-4 border-primary">Maintain</span>
+      </h1>
+      <ul className="text-2xl sm:p-2 p-2">
+        <li>
+          <a href="#">Home</a>
```
Comment (osamaakb) [inline src/components/Footer/Footer.js:18]: you should use links here to navigate through the router, checkout Navbar branch for more details

File: src/components/Card/LandingPageCard/LandingPageCard.js [hunk 3]
```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import propTypes from 'prop-types';
+import { motion } from 'framer-motion';
```
Comment (osamaakb) [inline src/components/Card/LandingPageCard/LandingPageCard.js:3]: did you make changes to the landing page card in this branch or these are old changes?

Comment (mahmodlte) [reply src/components/Card/LandingPageCard/LandingPageCard.js:3]: Yes i have used the primary color and changed animation to Framer Motion

Comment (osamaakb) [reply src/components/Card/LandingPageCard/LandingPageCard.js:3]: you should have changed these in the landingPage branch and pushed, however will check and merge them for now but in. the future make the changes in their branch its better to organize everything

File: src/components/Card/LandingPageCard/LandingPageCard.js [hunk 4]
```diff
@@ -1,15 +1,22 @@
...
   return (
-    <>
-      <div className=" max-h-lg max-w-sm sm:w-1/2 lg:w-1/4  overflow-hidden shadow-lg transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-105 rounded">
-        <img className="w-full" src={image} alt="Train to Maintain landing" />
-        <div className="font-bold text-xl text-purple-600 text-center p-10">
-          {title}
-        </div>
+    <motion.div
+      whileHover={{ scale: 1.02 }}
+      className=" max-h-lg max-w-sm sm:w-1/2 lg:w-1/4 overflow-hidden shadow-2xl  "
```
Comment (osamaakb) [inline src/components/Card/LandingPageCard/LandingPageCard.js]: dimension still not correct in small or even large it looks a square more than a rectangular, I suggest you make it fixed PX

File: src/components/Footer/Footer.js [hunk 5]
```diff
@@ -1,7 +1,49 @@
 import React from 'react';
+import workoutImage from './assets/workoutImage.jpg';
+import { Link } from 'react-router-dom';
 
 const Footer = () => {
-  return <div>Footer</div>;
-};
+  return (
+    <div className=" sm:h-auto lg:h-64 flex lg:flex-row xl:flex-row flex-col md:flex-row justify-between items-center font-poppins text-secondary">
```
Comment (osamaakb) [inline src/components/Footer/Footer.js]: No need for font-poppin here
```suggestion
    <div className=" sm:h-auto lg:h-64 flex lg:flex-row xl:flex-row flex-col md:flex-row justify-between items-center text-secondary">
```

**Pull request merged**